examples/vmm_v2/vms_clone_v2 .yml

```diff
@@ -0,0 +1,36 @@
+---
+# Summary:
+# This playbook will do:
+# 1. Clone VM with same attributes values
+# 2. Clone VM with different attributes values
+
+- name: VM clone playbook
+  hosts: localhost
+  gather_facts: false
+  module_defaults:
+    group/nutanix.ncp.ntnx:
+      nutanix_host: <pc_ip>
+      nutanix_username: <user>
+      nutanix_password: <pass>
+      validate_certs: false
+  tasks:
+    - name: Setting Variables
+      ansible.builtin.set_fact:
+        vm_uuid: "a990cfaa-95a8-4861-bdf6-14060555442d"
+
+    - name: Clone VM with same attributes values
+      nutanix.ncp.ntnx_vms_clone_v2:
+        ext_id: "{{ vm_uuid }}"
+        name: "integration_test_vm_clone1"
+      register: result
+      ignore_errors: true
+
+    - name: Clone VM with different attributes values
```

Comment:
Can we provide disk and all details if yes please include example for that as well